### PR TITLE
[FLINK-31554][docs-zh] Translate "Overview" page of "sql-gateway" into Chinese

### PR DESCRIPTION
**Pull request by FangYongs**
## What is the purpose of the change

This PR aims to translate "Overview" page of "sql-gateway" into Chinese shammon FY A minute ago

## Brief change log
  - Translate overview.md of sql-gateway into chinese

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no) no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no) no
  - The serializers: (yes / no / don't know) no
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know) no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / no / don't know) no
  - The S3 file system connector: (yes / no / don't know) no

## Documentation

  - Does this pull request introduce a new feature? (yes / no) no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/content.zh/docs/dev/table/sql-gateway/overview.md [hunk 1]
```diff
@@ -27,35 +27,33 @@ under the License.
 Introduction
```
Comment (fsk119) [inline docs/content.zh/docs/dev/table/sql-gateway/overview.md:27]: 我感觉每一小节的标题也可以翻译一下？以及第二行整个页面的标题

**Review by fsk119 — COMMENTED** [review]
Thanks for your contribution！

**Comment by FangYongs** [comment]
Thanks @fsk119 DONE

**Review by fsk119 — APPROVED** [review]
LGTM

**Pull request merged**